README.md

```diff
@@ -47,8 +63,6 @@ Please refer to the [Variables](#variables) section for required variables.
 
 ## Variables<a id="variables"></a>
```

Comment:
Maybe we should consider to put a action list here first as all the variable will be binding with action to be used.
And we might remove the duplicated definition for the variable items.